client/source/App.jsx

```diff
@@ -18,6 +23,11 @@ const App = function CreateAppComponent() {
     <div>
       <Header />
       <Overview />
+      <EmptyStar />
+      <QuarterStar />
+      <HalfStar />
+      <ThreeQuarterStar />
+      <FullStar />
```

Comment:
Probably shouldn't have these just chilling here in the `main` production branch.